backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraSingleton.java

```diff
@@ -19,15 +19,32 @@
 
 package org.apache.james.backends.cassandra;
 
+import java.util.concurrent.atomic.AtomicInteger;
+
 public class DockerCassandraSingleton {
+    private static final int MAX_TEST_PLAYED = 500;
+
+    private static final AtomicInteger testsPlayedCount = new AtomicInteger(0);
 
     public static final DockerCassandra singleton = new DockerCassandra();
 
     static {
         singleton.start();
     }
 
-    public static void restart() {
+    public static void incrementTestsPlayed() {
+        testsPlayedCount.incrementAndGet();
```

Comment:
what's the point of using an AtomicInteger as the restart is obviously not thread-safe